test/migrations/rid_db_post_migration_e2e.sh

```diff
@@ -82,7 +82,7 @@ docker run --link dummy-oauth-for-testing:oauth \
 	--link core-service-for-testing:core-service \
 	-v "${RESULTFILE}:/app/test_result" \
 	-w /app/monitoring/prober \
-	interuss/monitoring:v0.3.0 \
```

Comment:
Can we jump straight to v0.5.0?

Reply:
My bad, indeed, we want that otherwise we break things.